pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -634,6 +638,59 @@ func init() {
 	}
 
 	err = InfraSAs.addServiceAccount(
+		InfraPetSetControllerServiceAccountName,
+		authorizationapi.ClusterRole{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: PetSetControllerRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				// PetSetController.podCache.ListWatch
```

Comment:
It didn't require this to work, I'm again just following the pattern in this file which is agnostic of shared informer.